build/configure.js

```diff
@@ -26,22 +27,23 @@ function getLocale(grunt, options) {
 function buildRules(grunt, options, commons, callback) {
 	var locale = getLocale(grunt, options);
 	options.getFiles = false;
-	buildManual(grunt, options, commons, function (result) {
-
+	buildManual(grunt, options, commons, function(result) {
 		function parseMetaData(source, propType) {
-			var data = source.metadata
-			var key = source.id || source.type
+			var data = source.metadata;
+			var key = source.id || source.type;
 			if (key && locale && locale[propType] && propType !== 'checks') {
-				data = locale[propType][key] || data
+				data = locale[propType][key] || data;
 			}
 			var result = clone(data) || {};
 
 			if (result.messages) {
-				Object.keys(result.messages).forEach(function (key) {
+				Object.keys(result.messages).forEach(function(key) {
 					// only convert to templated function for strings
 					// objects handled later in publish-metadata.js
 					if (typeof result.messages[key] !== 'object') {
-						result.messages[key] = dot.template(result.messages[key]).toString();
+						result.messages[key] = dot
+							.template(result.messages[key])
```

Comment:
how does Prettier reconcile this change with moving `.forEach` here https://github.com/dequelabs/axe-core/pull/784/files#diff-35b4a816e0441e6a375cd925af50752cR322

Reply:
I'm not sure what you're asking.

Reply:
in the one case it moved the `.forEach` up onto the same line, in this case it moves the `.template` down - what is the logic?

Reply:
It's all about line-length. If it can't fit everything on 80 characters, it'll add a newline. If it can merge statements onto a single line, it will.